apps/example-todo-app/src/app/edge/route.ts

```diff
@@ -0,0 +1,15 @@
+import { withRouteSpecEdge } from "@/lib/middlewares"
+import { z } from "zod"
+
+export const runtime = "edge"
+
+const route_spec = {
+  jsonResponse: z.object({
+    return: z.boolean(),
+  }),
+  auth: "none",
+} as const
+
+export const GET = withRouteSpecEdge(route_spec)((req) => {
+  return req.responseEdge.status(200).json({ return: true })
```

Comment:
maybe mirroring the interface of non-edge with
```suggestion
export const GET = withRouteSpecEdge(route_spec)((req, res) => {
  return res.status(200).json({ return: true })
```
would be a bit easier to use (since `responseEdge` is already an artificial construct anyways)?

Reply:
I just created a PR to solve this, great idea.

https://github.com/seamapi/nextlove/pull/85

this will also make it easier to remove the duplicated code

https://github.com/seamapi/nextlove/pull/86